<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigSetDig</class>
 <widget class="QWidget" name="ConfigSetDig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="4,2">
       <item>
        <widget class="QGroupBox" name="item_Box">
         <property name="title">
          <string>当前标签</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>标签名：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="item_remark_edit"/>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>标签备注:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="item_explain"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>属性列表：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="attribute_list">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <column>
             <property name="text">
              <string>属性</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>备注名</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>标签数据类型:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="data_type">
            <item>
             <property name="text">
              <string>文本</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>数字</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>布尔</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>无</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="item_ok_btn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>修改</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>文件</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>文件名:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="file_remark_edit"/>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>文件备注:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="file_explain"/>
          </item>
          <item>
           <widget class="QLabel" name="file_path_label">
            <property name="text">
             <string>文件路径:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,1">
            <item>
             <widget class="QLabel" name="config_path_label">
              <property name="text">
               <string>配置文件路径:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>更改</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok_btn">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_btn">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
